rmgpy/molecule/atomtype.py

```diff
@@ -633,6 +696,27 @@ def get_features(self):
 ATOMTYPES['Sibf'].set_actions(increment_bond=[], decrement_bond=[], form_bond=[], break_bond=[], increment_radical=[], decrement_radical=[], increment_lone_pair=[], decrement_lone_pair=[])
 ATOMTYPES['Siq'].set_actions(increment_bond=[], decrement_bond=['Sit'], form_bond=[], break_bond=[], increment_radical=[], decrement_radical=[], increment_lone_pair=[], decrement_lone_pair=[])
 
+ATOMTYPES['P'].set_actions(increment_bond=['P'], decrement_bond=['P'], form_bond=['P'], break_bond=['P'], increment_radical=['P'], decrement_radical=['P'], increment_lone_pair=['P'], decrement_lone_pair=['P'])
+ATOMTYPES['P0sc'].set_actions(increment_bond=[], decrement_bond=[], form_bond=['P0sc'], break_bond=['P0sc'], increment_radical=['P0sc'], decrement_radical=['P0sc'], increment_lone_pair=[], decrement_lone_pair=['P1s', 'P1sc'])
+ATOMTYPES['P1s'].set_actions(increment_bond=['P1dc'], decrement_bond=[], form_bond=['P1s'], break_bond=['P1s'], increment_radical=['P1s'], decrement_radical=['P1s'], increment_lone_pair=['P0sc'], decrement_lone_pair=['P3s'])
+ATOMTYPES['P1sc'].set_actions(increment_bond=['P1dc'], decrement_bond=[], form_bond=['P1sc'], break_bond=['P1sc'], increment_radical=['P1sc'], decrement_radical=['P1sc'], increment_lone_pair=['P0sc'], decrement_lone_pair=['P3s'])
```

Comment:
Not sure whether this will be a problem. Since `P3sc` is no longer included, `decrement_lone_pair=['P3s']` for atom type `P1sc`. But `P1sc` has a charge of -1, while `P3s` has a charge of 0. It seems to me that the charge will be conflicted. I guess we can either keep the `P3sc` or modify `P3s`'s charge attribute?

Reply:
Okay, it seems like the charge will be updated, after applying `decrement_lone_pair()`

Reply:
I was looking at Nitrogen and Sulfur AtomType, and they have something similar to that (charged AtomType has non-charged AtomType for `decrement_lone_pair()` and `increment_lone_pair()`), so I thought it would be alright. 
So it's okay as it is?